src/main/webapp/app/admin/report-management/team-settings/team-config-dialog.controller.js

```diff
@@ -29,27 +30,51 @@
         function clear() {
             $uibModalInstance.close();
         }
+        function checkWebhhokIsExiste(webhook) {
+            for (var index = 0; index < vm.webhookList.length; index++) {
+                if (webhook.id) {
+                    if (webhook.webhookName === vm.webhookList[index].webhookName && webhook.id !== vm.webhookList[index].id) {
```

Comment:
this is not correct..use this validation in notification center to check duplicacy